src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -26,6 +28,9 @@
     {
         persons = new UniqueGuestList();
     }
+    private final UniqueRoomList rooms; {
+        rooms = new UniqueRoomList();
```

Comment:
put line 31 together with the declaration of persons, and put line 32 inside the block (line 28-30)

Reply:
Done.